test/integration/xds_integration_test.cc

```diff
@@ -266,6 +266,10 @@ class LdsInplaceUpdateHttpIntegrationTest
     config_helper_.addListenerFilter(tls_inspector_config);
     config_helper_.addSslConfig();
     config_helper_.addConfigModifier([](envoy::config::bootstrap::v3::Bootstrap& bootstrap) {
+      bootstrap.mutable_static_resources()
+          ->mutable_listeners(0)
+          ->mutable_connection_balance_config()
+          ->mutable_exact_balance();
```

Comment:
Can you make this a setup param and test this both ways (default off is fine just want to have at least 1 test where we have the other config.)